src/app/wallet/wallet.component.html

```diff
@@ -155,7 +155,7 @@ <h4 class="init-left__title">Your private, secure wallet</h4>
                                                 <span class="info__state-failed" *ngIf="transaction.state === 'FAILED'">
                                                     Failed
                                                 </span>
-                                                {{ transaction.date | date: "medium" }}
+                                                {{ transaction.date | date: 'medium' }}
```

Comment:
```suggestion
                                                {{ transaction.date | date:'medium' }}
```

Reply:
This was fixed in the last commit (https://github.com/AMPnet/ampnet-crowdfunding-frontend/pull/152/commits/df33199816a7d0a612d26a23335a8005518b6a2c)